src/lib/sidenav/sidenav.ts

```diff
@@ -398,6 +401,8 @@ export class MdSidenavLayout implements AfterContentInit {
   }
 
   _closeModalSidenav() {
```

Comment:
I would create a new method _onBackdropClicked that emits and then calls this, we will probably add other bindings that call this function later (e.g. esc key).